scripts/feed.js

```diff
@@ -14,11 +14,17 @@ function Feed(feed_urls)
 
     for(id in this.feed_urls){
       var archive = new DatArchive(this.feed_urls[id]);
-      var fileEvents = archive.createFileActivityStream()
+      try {
+        fileEvents = archive.createFileActivityStream();
```

Comment:
For some reason even with a try catch around this it still throws an uncaught error. Maybe I'm doing something wrong?

Reply:
Could it be an async issue with catch? I think archive.createActivityStream is mean to be into async

Reply:
Ah good point, probably